ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -452,6 +452,17 @@ else
 		camera_feedback start
 	fi
 
+	# Camera capture driver
+	if param greater CAM_CAP_FBACK 0
+	then
+		if camera_capture start
+		then
+			set FMU_MODE pwm4
```

Comment:
Work should be done on the px4fmu to support the mode needed on fmu V5.

https://github.com/PX4/Firmware/blob/master/src/drivers/px4fmu/fmu.cpp#L3048-L3050

The script should select that mode.